api/art/index.ts

```diff
@@ -0,0 +1,17 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import path from 'path';
```

Comment:
this should probably be node:path
```suggestion
import path from 'node:path';
```